src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -193,4 +193,20 @@ public void waitForOtherOptionTextToBeClickable(int qnNumber, int responseNumber
         waitForElementToBeClickable(element);
     }
 
+    public void verifyVisibilityAndCloseMoreInfoAboutEqualShareModal() {
+        WebElement moreInfoAboutEqualShareModalLink = browser.driver.findElement(By.id("more-info-equal-share-modal-link"));
+        click(moreInfoAboutEqualShareModalLink);
+        WebElement moreInfoAboutEqualShareModal = browser.driver.findElement(By.id("more-info-equal-share-modal"));
+        waitForElementVisibility(moreInfoAboutEqualShareModal);
+        closeMoreInfoAboutEqualShareModal();
+    }
+
+    public void closeMoreInfoAboutEqualShareModal() {
+        WebElement closeButton = browser.driver.findElement(By
+                .xpath("//div[@id='more-info-equal-share-modal']//button[@class='close']"));
```

Comment:
@damithc checked the parent class (`AppPage`). No such method exists. Should I create it? If so, should I create the method in the parent class or this class?